app/uk/gov/hmrc/plasticpackagingtax/returns/models/domain/ExportedPlasticWeight.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.returns.models.domain
+
+import play.api.libs.json.{Json, OFormat}
+
+case class ExportedPlasticWeight(totalKg: Long, totalValueForCreditInPence: Long) {
+  def totalKgAsString = this.totalKg.toString
+
+  def totalValueForCreditAsString =
+    BigDecimal(this.totalValueForCreditInPence / 100.00).setScale(2).toString
```

Comment:
this will not work as I think scala is using int constructor for this calculation
`this.totalValueForCreditInPence / 100.00` - the result is 0 (boxed to int)
what we need is this:
`(BigDecimal(this.totalValueForCreditInPence) / 100).setScale(2).toString`
here the result of this calculation: BigDecimal(this.totalValueForCreditInPence) / 100 is BigDecimal so allows floating point numbers.

Reply:
sorry, please test that, I'm not expert just was looking at same thing just now for my page.

Reply:
Yes!!!! Well spotted! :)

Reply:
Addressed.